src/app_updater.py

```diff
@@ -63,7 +65,23 @@ def _get_api_token(cls):
                 return api_token
         raise Exception('Max retries exceeded')
 
-    @classmethod
-    def _get_github_repository(cls):
-        api_token = cls._get_api_token()
-        return GithubRepository(api_token)
+    def _there_is_a_newer_version(self, github_repository: GithubRepository) -> bool:
```

Comment:
Cambiaria `there_is_a_newer_version` por `is_there_a_newer_version` o `validate_new_version` De esa forma me suena a mas sentido devolver true o false. 

Con _there_is_a_newer_version_  me da la sensacion de que se da por entendido que hay una nueva version

Reply:
mmm si, la verdad que justo en esta siempre me da dudas, por que lo ghe puesto siempre de ambas maneras, y de hecho creo que pense la tuya tambien.

Pero lo agregue asi por que se me hacia como mas legible en ingles plano la frase "if there is  a newer version"  como que el `if` le da la pregunta. y en ese caso entra.

En cambio " if is there a newer version" no es tan ingles plano, no se si se entiendo lo que quiero decir jaja

En resumen, creo que lo hice mas por una cuestion estetica al leer la frase completa junto al if